script/example.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+import yaml
+import json
+from argparse import ArgumentParser
+
+def action_json(args):
+
+    with open('scripts/example_data.json') as json_file:
+        print(json.dumps(json.loads(json_file.read()), indent=2))
+
+def action_yaml(args):
+    with open('scripts/example_data.yaml') as yaml_file:
+        print(yaml.dump(yaml.safe_load(yaml_file.read()), indent=2))
```

Comment:
It is recommended to use `yaml.safe_dump` instead of `yaml.dump` for security reasons, especially when dealing with untrusted input.
```suggestion
print(yaml.safe_dump(yaml.safe_load(yaml_file.read()), indent=2))
```